config/envs/local/config_ignore.settings.yml

```diff
@@ -0,0 +1,12 @@
+ignored_config_entities:
+  - 'system.theme:default'
+  - 'core.extension:theme'
+  - 'block.block.*'
+  - '~block.block.su_humsci_theme_*'
+  - '~block.block.su_humsci_admin_*'
+  - '~block.block.seven_*'
+  - 'bugherd_api.settings:project_key'
+  - system.site
```

Comment:
missing quotes?

Reply:
thats how it exported. i think it only needs the `'` if there is a `:` or `*`